arm-commerce/2015-06-01-preview/swagger/commerce.json

```diff
@@ -49,15 +49,15 @@
             "in": "query",
             "required": false,
             "type": "boolean",
-            "description": "When set to true (default), the aggregates are broken down into the instance metadata which is more granular."
+            "description": "This is an optional Boolean flag. `True` returns usage data in instance-level detail, `false` causes server-side aggregation with fewer details. For example, if you have 3 website instances, by default you will get 3 line items for website consumption. If you specify showDetails = false, the data will be aggregated as a single line item for website consumption within the time period (for the given subscriptionId, meterId, usageStartTime and usageEndTime)."
```

Comment:
It's not necessary to specifically state "This is an optional Boolean flag" -- it's implied with the `required: false` 

If you explicitly state it here, then everyone in every Azure service would always have to explicitly state that, and there would be chaos.  And perhaps a little weeping 😢